pattern/Disjunction.java

```diff
@@ -87,6 +87,7 @@ public boolean equals(Object o) {
         if (o == null || getClass() != o.getClass()) return false;
 
         Disjunction that = (Disjunction) o;
+        // TODO should use a set-comparison
```

Comment:
how do we think we should link these TODOs to an open issue? This is possible with #6115